### ISSUE TITLE
Market tooltip should not hide stars

### ISSUE THREAD

**Issue by froooze** [bitshares/bitshares-ui#2859]
**Describe the bug**
With the market tooltips the star icon is not accessible anymore. 

**To Reproduce**
1. Go to Exchange 
2. hover over a market

**Expected behavior**
Move the tooltip a little bit to the left side

**Screenshots**
![market-tooltip](https://user-images.githubusercontent.com/40874087/60395963-77fdac80-9b3b-11e9-9150-3d06562b8bf4.png)

**Comment by startailcoon** [comment]
I don't think we really need this, quite intrusive, tooltip on this place. It takes up to much space.
The reason is that there was an addition to the component used in multiple places to use the tooltip on assets.

**Comment by sschiessl-bcp** [comment]
I think a proper delay here will do.

**Comment by froooze** [comment]
#2809, but stars should not be hide by default
